### DISCUSSION TITLE
Emails won't stop!

### DISCUSSION THREAD

**Discussion by dpPandora** [community/community#34585] [New to GitHub]
Please!
BOTH of my inboxes are filled with github warning me that my personal tokens are expiring!
I make about 1 token a day which is probably bad practice but for now its w/e. So, you can imagine the spam github sends me. I've disabled all notifications in my profile but it still hasnt changed. It's even sending mail to my old primary email.
T~T I don't know what to do.

**Comment by jfullstackdev** [comment]
simply delete the token,

**Comment by DSDmark** [comment]
Also you can set no expire 
![Screenshot from 2022-10-06 14-06-49](https://user-images.githubusercontent.com/85744816/194264125-08364119-c8f9-4fc7-b143-b955e34ba627.png)
